### ISSUE TITLE
Add download icon to package links

### ISSUE THREAD

**Issue by nlhkabu** [pypa/warehouse#880]
as per the discussion here on PR https://github.com/pypa/warehouse/pull/878

I'd like to add a download icon to the project links. This code will need to be the first if statement::
```
{% if name|lower == "download" %}
   {% set icon = "cloud-download" %}
```

**Comment by umeshksingla** [comment]
I have submitted pull request for this issue. Please check #885 .

**Comment by dstufft** [comment]
Closed by #885

**Comment by nlhkabu** [comment]
Thanks @umeshksingla :D
